packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -701,8 +712,12 @@ async function createReferencePriceFeedForFinancialContract(
 }
 
 function getFinancialContractIdentifierAtAddress(web3, financialContractAddress) {
-  const ExpiringMultiParty = getTruffleContract("ExpiringMultiParty", web3, "1.2.0");
-  return new web3.eth.Contract(ExpiringMultiParty.abi, financialContractAddress);
+  try {
```

Comment:
if you provide this method with an unknown identifier or an invalid EMP it tends to throw a really nasty hard to debug error. As a result, I've wrapped it in a try catch to try and help with debugging these cases.